sklearn/kernel_approximation.py

```diff
@@ -245,9 +245,8 @@ class AdditiveChi2Sampler(BaseEstimator, TransformerMixin):
     References
     ----------
     See `"Efficient additive kernels via explicit feature maps"
-    <http://eprints.pascal-network.org/archive/00006964/01/vedaldi10.pdf>`_
+    <http://citeseer.ist.psu.edu/viewdoc/summary?doi=10.1.1.167.7024>`_
```

Comment:
I thought citeceerx would be more stable and the link would still be alive even if anything happens to the external PDF link like what happened now...

But yes the extended journal link would be better to have though! I think we could link the journal version and have https://github.com/scikit-learn/scikit-learn/issues/4344#issuecomment-77744354 for checking all our links!!